README.md

```diff
@@ -1,13 +1,55 @@
 # Catalog CSV Import module
 Catalog CSV Import module
 
+
+
 # Installation
 Installing the module:
 * Automatically: in VC Manager go to Configuration -> Modules -> Catalog CSV Import module -> Install
 * Manually: download module zip package from https://github.com/VirtoCommerce/vc-module-catalog-csv-import/releases. In VC Manager go to Configuration -> Modules -> Advanced -> upload module package -> Install.
 
 # Settings
 
+Before starting using the Import functionality, the admin should set the 'Create new dictionary values' for Import Module. In order to set this parameter, the user should go through the following steps:
+
+1. Open Settings→select CSVCatalogImport→select General;
+1. The system will open the 'Settings values' blade;
+1. The user switches on  the 'Create new dictionary value' switch and saves the changes;  
+1. The system will save the changes and enable creation of new Property Dictionary Values during data import.
+
+![Fig. Settings](docs/media/screen-settings.png)
+![Fig. Setting values](docs/media/screen-setting-values.png)
+
+## Catalog Import
+
+### Scenario
+
+1. The user opens Catalog module→ selects a Catalog and clicks 'Import';
+![Catalog Import](docs/media/screen-select-catalog-to-import.png)
+1. The system will open the 'Catalog.blades.importers-list.title' blade and inform the user that the import should be in CSV format;
+1. The user opens the next blade
+The system will prompt the user to do the following:
+    1. Select delimeter from drop down;
+         1. Semicolon should be selected only;
```

Comment:
I think it is redundant info (about default selection), it could be treated as Semicolon should be selected for starting import.